cmd/fyne/internal/commands/flags.go

```diff
@@ -36,3 +38,191 @@ func (k *keyValueFlag) String() string {
+		return &cli.StringFlag{
+			Name:        "developer",
+			Aliases:     []string{"dev"},
+			Usage:       "windows: the developer identity for your Microsoft store account",
+			Destination: dst,
+		}
+	},
+	"executable": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "executable",
+			Aliases:     []string{"exe"},
+			Usage:       "set path to executable",
+			DefaultText: "current directory main binary",
+			Destination: dst,
+		}
+	},
+	"icon": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "icon",
+			Usage:       "set name of the application icon file",
+			Destination: dst,
+		}
+	},
+	"install-dir": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "install-dir",
+			Aliases:     []string{"o"},
+			Usage:       "specify location to install to, rather than the OS default",
+			Destination: dst,
+		}
+	},
+	"key-name": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-name",
+			Usage:       "android: alias for the signer's private key, which is needed when reading a .keystore file",
+			Destination: dst,
+		}
+	},
+	"key-pass": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-pass",
+			Usage:       "android: password for the signer's private key, which is needed if the private key is password-protected",
+			DefaultText: "read from stdin",
+			Destination: dst,
+		}
+	},
+	"key-store": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-store",
+			Usage:       "android: location of .keystore file containing signing information",
+			Destination: dst,
+		}
+	},
+	"key-store-pass": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-store-pass",
+			Usage:       "android: password for the .keystore file",
+			DefaultText: "read from stdin",
+			Destination: dst,
+		}
+	},
+	"name": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "name",
+			Usage:       "set name of the application",
+			DefaultText: "executable file name",
+			Destination: dst,
+		}
+	},
+	"output": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "output",
+			Usage:       "specify name for the output file",
+			DefaultText: "based on current directory",
+			Destination: dst,
+		}
+	},
+	"password": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "password",
+			Aliases:     []string{"pass"},
+			Usage:       "windows: password for the certificate used to sign the build",
+			Destination: dst,
+		}
+	},
+	"profile": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "profile",
+			Usage:       "ios/macos: name of the provisioning profile for this release build",
+			Destination: dst,
+			Value:       "XCWildcard",
+		}
+	},
+	"source-dir": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "source-dir",
+			Aliases:     []string{"src"},
```

Comment:
I wonder if sticking to only `src` would be fine? I see little reason to use the long one.

Reply:
Agreed, seems cleaner.